jwst/pixel_replace/tests/test_pixel_replace.py

```diff
@@ -238,3 +244,84 @@ def test_pixel_replace_nirspec_ifu(input_model_function, algorithm):
 
     result.close()
     input_model.close()
+
+
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize('input_model_function',
+                         [nirspec_ifu])
+@pytest.mark.parametrize('algorithm', ['fit_profile', 'mingrad'])
+def test_pixel_replace_nirspec_ifu_container_names(tmp_cwd, tmp_path, input_model_function, algorithm):
+    """
+    Test pixel replacement for NIRSpec IFU using a container
+
+    Larger data and more WCS operations required for testing make
+    this test take more than a minute, so marking this test 'slow'.
+
+    The test is otherwise the same as for other modes.
+    """
+    output_dir = tmp_path / 'output'
+    output_dir.mkdir(exist_ok=True)
+    output_dir = str(output_dir)
+    
+    input_model, bad_idx = input_model_function()
+    input_model.meta.filename = 'jwst_nirspec_1_cal.fits'
+    input_model2, bad_idx2 = input_model_function()
+    input_model2.meta.filename = 'jwst_nirspec_2_cal.fits'
+    cfiles = [input_model, input_model2]
+    container = ModelContainer(cfiles) 
+
+    expected_name = []
+    expected_name.append('jwst_nirspec_1_pixelreplacestep.fits')
+    expected_name.append('jwst_nirspec_2_pixelreplacestep.fits')
+
+    return_files = []
+    # for this simple case, the results from either algorithm should
+    # be the same
+    result = PixelReplaceStep.call(container, skip=False, algorithm=algorithm,save_results=True)
+    for dirname in [output_dir, tmp_cwd]:
```

Comment:
Why do we need to check both the cwd and the output dir here?  I would have thought we would know beforehand which files went to which directories, and part of the test would be to ensure those are as expected.